modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -65,16 +65,39 @@ spec:
 <4> Specify the `kafka` type.
 <5> Specify the URL and port of the Kafka broker as a valid absolute URL, optionally with a specific topic. You can use the `tcp` (insecure) or `tls` (secure TCP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
 <6> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
```

Comment:
shared_key is required for fluentforward protocol https://github.com/openshift/cluster-logging-operator/blob/master/pkg/generators/forwarding/fluentd/templates.go#L660

Reply:
Hi, @jcantrill. Would you take a look at the current [Forwarding logs using the legacy Fluentd method](https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-external.html#cluster-logging-collector-legacy-fluentd_cluster-logging-external) topic to see if its coverage of `shared_key` can be improved? If so, please create a new JIRA for that issue and assign it to me. Thanks.